Dockerfile

```diff
@@ -67,6 +72,7 @@ RUN apt-get update && apt-get install -y \
 	xfsprogs \
 	libzfs-dev \
 	tar \
+	linux-image-`uname -r` \
```

Comment:
I meant does it work on arch with custom kernel, not sure we should support custom kernel maybe :( but if it fixable without hacks I guess it's fine